python/pyrogue/gui/variables.py

```diff
@@ -187,7 +187,7 @@ def varListener(self, path, value):
                     self.updateGui.emit(i)
             elif isinstance(self._widget, QSpinBox):
                 if self._widget.value != value.value:
-                    self.updateGui.emit(value)
+                    self.updateGui.emit(value.value)
```

Comment:
`value.value` is a little strange.  Maybe the `def varListener(self, path, value):` should be changed to `def varListener(self, path, var):` then it would be `var.value`, which is more intuitive when reading the code